### PR TITLE
Ot153 39 organization details screen

### PR DESCRIPTION
**Pull request by frangdelsolar**
## JIRA
[OT153-39](https://alkemy-labs.atlassian.net/browse/OT153-39)

## Short description

Added OrganizationDetailsComponent: view, logic, routing.

## Screenshot or Video

image

### Sanity Check Steps

- [x] All console logs have been removed.
- [x] This change was tested on my localhost.
- [ ] This change depends for another ticket.
- [x] All linked JIRA acceptance criteria are satisfied.
- [ ] Does this change require environment variables.

### CODE REVIEW

File: src/app/shared/models/organization.model.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export class Organization{
```
Comment (RicardoMaldonado93) [inline src/app/shared/models/organization.model.ts:1]: No poner toda la interfaz como opcional, ya que es practicamente poner lo mismo que any, poner como obligatorios aquellos datos que son requeridos, tales como el ID, name, logo, etc

File: src/app/features/pages/organization-details/organization-details.component.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import { Organization } from 'src/app/shared/models/organization.model';
+
+@Component({
+  selector: 'app-organization-details',
+  templateUrl: './organization-details.component.html',
+  styleUrls: ['./organization-details.component.scss']
+})
+export class OrganizationDetailsComponent implements OnInit {
+
+  organization: Organization;
```
Comment (RicardoMaldonado93) [inline src/app/features/pages/organization-details/organization-details.component.ts:11]: Por favor arreglar esto 
image

File: src/app/features/pages/organization-details/organization-details.component.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<div class="container">
+<p-card header="{{organization.name}}" subheader="{{organization.short_description}}" [style]="{width: '360px'}" styleClass="p-card-shadow">
```
Comment (RicardoMaldonado93) [inline src/app/features/pages/organization-details/organization-details.component.html:2]: Centrar la tarjeta en la pantalla para que tenga un mejor aspecto

**Review by RicardoMaldonado93 — COMMENTED** [review]
@frangdelsolar Please fix the observations and update the branch to continue with the PR

**Comment by frangdelsolar** [comment]
@RicardoMaldonado93 

Short description
Code refactoring. Styling modified. Develop rebase

image

Sanity Check Steps
[x] All console logs have been removed.
[x] This change was tested on my localhost.
[] This change depends for another ticket.
[x] All linked JIRA acceptance criteria are satisfied.
[] Does this change require environment variables.

**Review by RicardoMaldonado93 — APPROVED** [review]
Good job! :tada:

**Pull request merged**